packages/xo-server/src/xapi/mixins/vm.js

```diff
@@ -454,6 +454,8 @@ export default {
         this.startVm(vm.$id)::ignoreErrors()
       } else if (vm.power_state === 'Suspended') {
         this.resumeVm(vm.$id)::ignoreErrors()
+      } else if (vm.power_state === 'Paused') {
+        this.pauseVm(vm.$id)::ignoreErrors()
```

Comment:
Does it make sense?

Can we pause an halted VM?

Reply:
I can't see that in the VM view (I can't pause a halted VM). But in the home/VM view, if I do that with grouped actions, I'll end with a `VM_BAD_POWER_STATE` error message in Settings/logs

Reply:
That's a typo, we want to *un*pause the VM if it's Paused.